Sources/OpenAPIRuntime/Base/OpenAPIValue.swift

```diff
@@ -139,9 +139,9 @@ public struct OpenAPIValueContainer: Codable, Equatable, Hashable, Sendable {
             try container.encode(value)
         case let value as String:
             try container.encode(value)
-        case let value as [OpenAPIValueContainer?]:
+        case let value as [Sendable?]:
             try container.encode(value.map(OpenAPIValueContainer.init(validatedValue:)))
-        case let value as [String: OpenAPIValueContainer?]:
+        case let value as [String: Sendable?]:
```

Comment:
```suggestion
        case let value as [String: (any Sendable)?]:
```

And elsewhere

Reply:
Ah good reminder, I opened #26 to land that first, then I'll update this PR.